CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 **SDK**
 - **Deprecated:** Old `getUniqueId`interface.
 - **Feature:** Added `getUniqueId` new interface for invoking data using `onSuccess` and `onError`.
+- **Feature:** Support `mailto` uri.
```

Comment:
Maybe we can elaborate it more as following:

Added support for URI's using `mailto` e.g. `<a href="mailto:test@example.com">send email</a>` for the MiniApps.

Reply:
updated with the referral link.